Sources/BraveWallet/Extensions/BraveWalletSwiftUIExtensions.swift

```diff
@@ -97,6 +97,14 @@ extension BraveWallet.BlockchainToken: Identifiable {
     // since there is no swap support for custom networks.
     return symbol == network.symbol ? BraveWallet.ethSwapAddress : contractAddress
   }
+  
+  var nftTokenTitle: String {
+    if isErc721, let tokenId = Int(tokenId.removingHexPrefix, radix: 16) {
+      return "\(name) #\(tokenId)"
+    } else {
+      return name
+    }
+  }
 }
```

Comment:
This should be in `BraveWalletExtensions` instead of `BraveWalletSwiftUIExtensions`. i have this in my erc721 PR. can you rebase after i merge?